### ISSUE TITLE
Configure sampling for service that processes 1.5 billion requests per day

### ISSUE THREAD

**Issue by chinarosesz** [Microsoft/ApplicationInsights-aspnetcore#718]
I'm unsure if I should use Adaptive sampling or Fixed-rate sampling for a service that processes about 2 billion requests per day. 

If I set a fixed rate of 1%, that would be 20 million events per day, I think that's still too much. If I use adaptive sampling, I'm not sure what is the appropriate value to set for max percentage and min percentage as the number of requests per day can vary between 1.5 to 2 billion. Do you have any recommendations on how I should approach this?

I wrote a sample application to send 100 exceptions to Application Insights, if I set the initial percentage to be .1%, nothing shows up, but if I set it 1%, there is one sample data, with the accurate count of 100 events.

**Comment by SergeyKanzhelev** [comment]
is it 2 billion from 1 server?

You can set fixed rate with the number smaller than 1. Minimal number you can set on SDK is `1E-6`: https://github.com/Microsoft/ApplicationInsights-dotnet/blob/c8a5d0b3cd6381b82bd1f94a784b87c0e107eb4a/src/ServerTelemetryChannel/BackwardsCompatability/SamplingPercentageEstimatorSettings.cs#L193

**Comment by chinarosesz** [comment]
Discussed offline with Sergey. This is now resolved.

I decided to use fixed rate sampling for this specific scenario since I know how much traffic is expected each day. Also, during testing, the statistics is a little bit off because sample data is smaller than in production. Count of requests will be more accurate when reported to Application Insights as the service continues to process for requests continuously compared to testing on local machine using small set of data.
